Dockerfile

```
@@ -123,7 +123,7 @@ RUN apt-get install -y --no-install-recommends --no-install-suggests wget && \
 RUN cp -r /ceres_installed/* /usr/local/
 
 # Build pyceres.
-RUN git clone --depth 1 --recursive https://github.com/cvg/pyceres
+RUN git clone --depth 1 -b v2.3 --recursive https://github.com/cvg/pyceres
```

This will fail, probably I need to also install Colmap from source explicitly here. With the latest version of pyceres this wasn't needed.
```suggestion
RUN git clone --depth 1 -b v1.0 --recursive https://github.com/cvg/pyceres
```